index.html

```diff
@@ -0,0 +1,298 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="stylesheet.css"/>
+  <title>Desktop_Portfolio</title>
+</head>
+<body>
+  <body>
+    <header id="header">
+      <div class="logo_container">
+        <a href="/">
+          <img class="my_logo" src="images/MyLogo.png">
```

Comment:
- [ ]  Kyndly make sure the images appear on your deployed web app 

![image](https://user-images.githubusercontent.com/95243996/200337999-fb0208a6-0dfb-4ddc-87dc-1fca0d58a38a.png)

![image](https://user-images.githubusercontent.com/95243996/200338066-c0f26917-a17c-404d-974d-10827f99460b.png)

## Note: This may be caused by the way you are referencing the src on the img tag.

![image](https://user-images.githubusercontent.com/95243996/200338479-85031cc2-816c-47ca-9ad8-dd5384a2f88b.png)

Try using `src="./images/MyLogo.png"` notice that the dot references the external folder Images